src/eru.Application/Common/Interfaces/IMessageService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using eru.Domain.Entity;
+
+namespace eru.Application.Common.Interfaces
+{
+    public interface IMessageService
+    {
+        Task<IEnumerable<string>> GetIdsOfAllSubscribers();
+        Task<IEnumerable<string>> GetIdsOfSubscribersInClass(Class targetClass);
+        Task SendSubstitutionNotification(string idOfTarget, Substitution substitution);
```

Comment:
```suggestion
        Task SendSubstitutionNotification(string idOfTarget, Substitution substitution);
        Task SendGenericNotification(string idOfTarget, string content)
```

There should be a method to send a generic notification.